src/main/native/com/code_intelligence/jazzer/driver/BUILD.bazel

```diff
@@ -35,7 +35,10 @@ cc_library(
     name = "coverage_tracker",
     srcs = ["coverage_tracker.cpp"],
     hdrs = ["coverage_tracker.h"],
-    deps = ["//src/main/java/com/code_intelligence/jazzer/runtime:coverage_map.hdrs"],
+    deps = [
+        "//src/main/java/com/code_intelligence/jazzer/runtime:coverage_map.hdrs",
+        "@jazzer_libfuzzer//:libfuzzer_no_main",
```

Comment:
Is this needed? I think that it ends up loading libFuzzer twice (once here and once in FuzzTargetRunnerNatives).